extensions/jackson/runtime/src/main/java/io/quarkus/jackson/runtime/ObjectMapperProducer.java

```diff
@@ -9,18 +9,25 @@
 import javax.enterprise.inject.Produces;
 import javax.inject.Singleton;
 
+import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.ObjectMapper;
 
 import io.quarkus.arc.DefaultBean;
+import io.quarkus.jackson.ObjectMapperCustomizer;
 
 @ApplicationScoped
 public class ObjectMapperProducer {
 
     @DefaultBean
     @Singleton
     @Produces
-    public ObjectMapper objectMapper(Instance<ObjectMapperCustomizer> customizers) {
+    public ObjectMapper objectMapper(Instance<ObjectMapperCustomizer> customizers,
+            JacksonConfigSupport jacksonConfigSupport) {
         ObjectMapper objectMapper = new ObjectMapper();
+        if (!jacksonConfigSupport.isFailOnUnknownProperties()) {
```

Comment:
Good idea to have this first, as it allows users to override the behavior with their own customizers if need be